keras_cv/models/object_detection/faster_rcnn.py

```diff
@@ -382,7 +388,7 @@ def __init__(
             samples_per_image=256,
             positive_fraction=0.5,
         )
-        self.nms_decoder = nms_decoder or NMSDecoder()
+        self._prediction_decoder = NMSDecoder(bounding_box_format=bounding_box_format)
```

Comment:
Shouldn't this continue to use `prediction_decoder` instead of the default, when specified?

Reply:
Good catch. Done